cpu/atmega_common/periph/i2c.c

```diff
@@ -226,6 +227,7 @@ void i2c_acquire(i2c_t dev)
 {
     assert(dev < I2C_NUMOF);
 
+    pm_block(4); /* Require clkIO */
```

Comment:
I feel like these should be protected by the mutex for example:

task 1 - block pm
task 1 - acquire i2c
task 2 - block pm
task 2 - waiting to acquire i2c
task 1 - release i2c
task 1 - unblock pm
task 2 - acquire i2c **(but pm is not blocked)**

Reply:
In that case pm_blockers would be set to 2 when both tasks do the `pm_block()`.
When task 1 unblocks pm, pm_blockers is still at 1, so pm is blocked when task 2 acquires the mutex.

Reply:
Ahh it counts... Good good. Thanks for the clarification.